dashboard/influx-data-gen/influx-data-gen

```diff
@@ -2,7 +2,7 @@
 source ../grafana-influxdb-telegraf-prometheus/configuration.env
 
 DOME_BUCKET="dome"
-INFLUX_API_URL="http://localhost:8086/api/v2/write?org=$DOCKER_INFLUXDB_INIT_ORG&bucket=$DOME_BUCKET&precision=ns"
+INFLUX_API_URL="http://192.168.0.9:8086/api/v2/write?org=$DOCKER_INFLUXDB_INIT_ORG&bucket=$DOME_BUCKET&precision=ns"
```

Comment:
is this IP part of the docker-compose setup?

Reply:
This was garbage. Just fixed!